tests/templates/kuttl/ldap-authentication/10-install-druid.yaml.j2

```diff
@@ -0,0 +1,68 @@
+---
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+metadata:
+  name: install-druid
+commands:
+  - script: |
+      kubectl apply -n $NAMESPACE -f - <<EOF
+      ---
+      apiVersion: druid.stackable.tech/v1alpha1
+      kind: DruidCluster
+      metadata:
+        name: derby-druid
+      spec:
+        image:
+          productVersion: "{{ test_scenario['values']['druid'].split('-stackable')[0] }}"
+          stackableVersion: "{{ test_scenario['values']['druid'].split('-stackable')[1] }}"
+        clusterConfig:
+          authentication:
+              - authenticationClass: druid-ldap-auth-class-$NAMESPACE
+          deepStorage:
+            hdfs:
+              configMapName: druid-hdfs
+              directory: /druid
+          metadataStorageDatabase:
+            dbType: derby
+            connString: jdbc:derby://localhost:1527/var/druid/metadata.db;create=true
+            host: localhost
+            port: 1527
+          zookeeperConfigMapName: druid-znode
+{% if lookup('env', 'VECTOR_AGGREGATOR') %}
+          vectorAggregatorConfigMapName: vector-aggregator-discovery
+{% endif %}
+        brokers:
```

Comment:
We could slightly reduce the verbosity here by using a yaml anchor once and referencing it for following roles.

Reply:
That's right, good catch. On the other hand i would like to keep it consistent and would rather not change all the druid tests if that is ok for you?

Reply:
There's also an argument to be made to keep the complexity visible, more understandable and changes more apparent. Completely fine by me.

Reply:
Alright thx!